constants/maro_networth/generators/networthGenerator.js

```diff
@@ -16,6 +16,23 @@ const getNetworth = async function (data, profile, bank) {
 
         for (const item of data[key].filter((i) => i.price)) {
             category.total += item.price;
+
+            // Looks pretty bad, I know.. if (item?.modified?.calculation?.length > 0) calculation: item.modified.calculation
```

Comment:
is it needed to not show the calculation if it doesn't exist?

Reply:
Not really